ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/network/DefaultP2PNetwork.java

```diff
@@ -224,6 +224,7 @@ public void start() {
                 }
                 dnsPeers.set(peers);
               });
+      dnsDaemon.start();
```

Comment:
:+1:  amazing that this has been missing since it was originally implemented

Reply:
we should add a test that ensures dnsDaemon is started if dnsDiscoveryURL is present